### PR TITLE
supprt runOnce

### PR DESCRIPTION
**Pull request by oren-codefresh**

### CODE REVIEW

File: agent/index.js [hunk 1]
```diff
@@ -141,6 +148,19 @@ class Agent {
...
+	_runOnce(Job) {
+		if (!runOnce) {
+			return;
+		}
+		const taskLogger = this.logger.child({
+			namespace: LOGGER_NAMESPACES.TASK,
+			job: Job.name,
+			uid: new Chance().guid(),
+		});
+		const job = new Job(this.codefreshAPI, this.kubernetesAPI, taskLogger);
```
Comment (olegsu) [inline agent/index.js]: You should pass `this.runtimes`, not `kubernetesAPI`

File: agent/index.js [hunk 2]
```diff
@@ -141,6 +148,19 @@ class Agent {
 		});
 	}
 
+	_runOnce(Job) {
+		if (!runOnce) {
```
Comment (olegsu) [inline agent/index.js]: Where this variable declared in this scope?

File: agent/index.js (hunk 1 above)
Comment (oren-codefresh) [reply agent/index.js]: done

File: agent/index.js (hunk 2 above)
Comment (oren-codefresh) [reply agent/index.js]: done

**Comment by olegsu** [comment]
LGTM

**Pull request merged**